homework7/src/main/java/raisetech/homework7/NameController.java

```diff
@@ -31,18 +32,16 @@ public ResponseEntity<String> createUser(@RequestBody @RequestParam(value = "nam
     URI url = builder.path(path).build().toUri();
 
     return ResponseEntity.created(url).body("name successfully created");
-
   }
 
   //PATCHメソッドはPOSTと同じく入力した文字列が空文字、null、20文字以上なら受け取らないで、URLパラメータの「names/」の後に整数を入れる仕様
   @PatchMapping("/names/{id}")
-  public ResponseEntity<Map<String, String>> updateUser(@PathVariable("id") String id,  @RequestBody @Validated UserForm form) {
-    return ResponseEntity.ok(Map.of("message", "name successfully updated"));
+  public Map<String, String> updateUser(@PathVariable("id") String id,  @RequestBody @Validated UserForm form) {
+    return Map.of("message", "name successfully updated");
   }
 
   @DeleteMapping("/names/{id}")
   public ResponseEntity<Map<String, String>> deleteUser(@PathVariable("id") String id) {

```

Comment:
こちらもupdateUserと同様にMapを返すようにしましょう。

Reply:
40行目の「Map.of」を「Map」に変更しました。

Reply:
40行目はMap.ofのままで良かったです！
44行目も同じようにMapを戻り値の型にしてほしいです！

```suggestion
  public Map<String, String> deleteUser(@PathVariable("id") String id) {

```

Reply:
前回の間違っていた変更（40行目の「Map.of」を「Map」に変更）を戻して、44,45行目の「ResponseEntity」を「Map」に変更しました。